### PR TITLE
Added kit type and version to the hw-info event

### PR DESCRIPTION
**Pull request by radujipa**
*NOTE:* I did the minimum amount of work here to move this task ahead. Ideally, we would think of a way to separate the event specs _coming in from/agreed upon with_ Vincent and generate the events in such a way as to not require other dependencies for the tracker, which is much larger in scope than the ticket itself. (If I get some time this sprint I might come back to it).

Requires: https://github.com/KanoComputing/kano-peripherals/pull/55
Requires: https://github.com/KanoComputing/kano-desktop/pull/263

### CODE REVIEW

**Comment by radujipa** [comment]
[(1) As a data scientist, I want to know if an OS is using CKC v1.1 [2]](https://trello.com/c/xOMEwPU7/1354-1-as-a-data-scientist-i-want-to-know-if-an-os-is-using-ckc-v11-2)

File: kano_profile/tracker/tracking_events.py [hunk 1]
```diff
@@ -15,10 +15,33 @@
...
+        kit_version = get_ck2_pro_version()
+
     track_data('hw-info', {
         'keyboard': 'kano' if detect_kano_keyboard() else 'generic',
         'model': get_rpi_model(),
-        'partitions': get_partition_info()
+        'partitions': get_partition_info(),
+        'kit': {
+            'type': kit_type,
+            'version': kit_version.vstring
```
Comment (tombettany) [inline kano_profile/tracker/tracking_events.py]: Isn't this going to fail for the `kit_version = None` case?

Comment (radujipa) [reply kano_profile/tracker/tracking_events.py]: Yes, of course it will..... :man_facepalming:

Comment (radujipa) [reply kano_profile/tracker/tracking_events.py]: Fixed.

**Pull request merged**